<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieViewWidget</class>
 <widget class="QWidget" name="MovieViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie View</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="contentWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>680</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <item>
        <widget class="QGroupBox" name="basicInfoGroup">
         <property name="title">
          <string>Informacje podstawowe</string>
         </property>
         <layout class="QFormLayout" name="basicInfoLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="titleRowLabel">
            <property name="text">
             <string>Tytuł:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Brak wybranego filmu</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold; font-size: 16px;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="yearRowLabel">
            <property name="text">
             <string>Rok produkcji:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="yearLabel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lengthRowLabel">
            <property name="text">
             <string>Długość:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lengthLabel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="genreRowLabel">
            <property name="text">
             <string>Gatunek:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="genreLabel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="ratingRowLabel">
            <property name="text">
             <string>Ocena:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="ratingLabel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="watchedRowLabel">
            <property name="text">
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="watchedLabel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="directorGroup">
         <property name="title">
          <string>Reżyser</string>
         </property>
         <layout class="QVBoxLayout" name="directorLayout">
          <item>
           <widget class="QLabel" name="directorLabel">
            <property name="text">
             <string>Brak informacji</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="descriptionGroup">
         <property name="title">
          <string>Opis</string>
         </property>
         <layout class="QVBoxLayout" name="descriptionLayout">
          <item>
           <widget class="QTextEdit" name="descriptionText">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="actorsGroup">
         <property name="title">
          <string>Obsada</string>
         </property>
         <layout class="QVBoxLayout" name="actorsLayout">
          <item>
           <widget class="QLabel" name="actorsLabel">
            <property name="text">
             <string>Brak informacji</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Edytuj film</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backButton">
           <property name="text">
            <string>Wróć</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
